tools/licenses/lib/main.dart

```diff
@@ -1964,6 +1982,8 @@ class _RepositoryRootThirdPartyDirectory extends _RepositoryGenericThirdPartyDir
       return _RepositoryPkgDirectory(this, entry);
     if (entry.name == 'vulkan')
       return _RepositoryVulkanDirectory(this, entry);
+    if (!entry.fullName.endsWith('externals/vulkanmemoryallocator'))
```

Comment:
Should this condition be the opposite? That is, if it *does* end with the string, then we *do* want to return `_RepositoryVulkanMemoryAllocatorDirectory`?

Reply:
D'oh, yes